src/database/models.py

```diff
@@ -2308,30 +2288,34 @@ class AboutUsPageSettings(models.Model):
   community = models.ForeignKey(Community, on_delete=models.CASCADE, db_index=True)
   title = models.CharField(max_length=LONG_STR_LEN, blank=True)
   sub_title = models.CharField(max_length=LONG_STR_LEN, blank=True)
-  description = models.TextField(max_length=LONG_STR_LEN, blank = True)
-  images = models.ManyToManyField(Media, blank=True)
-  featured_video_link = models.CharField(max_length=SHORT_STR_LEN, blank = True)
+  description = models.TextField(max_length=LONG_STR_LEN, blank=True)
+  # images = models.ManyToManyField(Media, blank=True)
+  image = models.ForeignKey(Media, blank=True, null=True, on_delete=models.SET_NULL)
```

Comment:
All the pageSettings models have possibilities for multiple images, which I would like to preserve even if not used currently.  Don't want this page settings to be different from all the others